contract/rust/src/lib.rs

```diff
@@ -20,13 +33,18 @@ pub struct Welcome {
 #[near_bindgen]
 impl Welcome {
     pub fn set_greeting(&mut self, message: String) {
+        // We can use env:: to access context from the blockchain transaction
         let account_id = env::signer_account_id();
         self.records.insert(account_id, message);
     }
 
+    // For users unfamiliar with Rust, this function is essentially returning our struct TextMessage defined above
```

Comment:
we could just contrast with `switch..case` statements which are very common in C-family languages.